pwiz_tools/Skyline/TestData/CommandLineTest.cs

```diff
@@ -853,6 +853,203 @@ private static void CheckRefSpectraAll(IList<DbRefSpectra> refSpectra)
             Assert.IsTrue(!refSpectra.Any());
         }
 
+        [TestMethod]
+
+        public void ConsoleAddAnnotationsTest()

```

Comment:
I would recommend moving this test into a separate class (e.g. "ConsoleAnnotationsTest" which inherits from "AbstractUnitTestEx") because CommandLineTest has gotten rather large.

Also, if there are steps in this test that do not depend on other steps in this test, then it would be better if they were separate [TestMethod] functions such as "ConsoleAddInvalidAnnotationTargetListTest" so that there is less code to step through when you are trying to debug a test failure.